src/main/resources/application.yaml

```diff
@@ -179,9 +179,11 @@ darts:
     arm-api:
       arm-username: ${ARM_USERNAME:}
       arm-password: ${ARM_PASSWORD:}
+      cabinet-id: 100
       url: ${ARM_URL:}
       token-path: /api/v1/token
       update-metadata-path: /api/v3/UpdateMetadata
+      download-data-path: /api/v1/downloadBlob/{cabinetId}/{recordId}/{fileId}
```

Comment:
I think our interpretation of [The HMCTS Way](https://hmcts.github.io/) API guidelines were to have the path parameters using `snake_case`.

So up to you whether you want to change them here - it's an internal system call, so not apparent in our darts-api swagger.

Reply:
I'd rather change it to follow the API guidelines, cheers